src/openforms/typing.py

```diff
@@ -14,19 +16,19 @@
 else:
     _StrOrPromise = str
 
-JSONPrimitive: TypeAlias = str | int | float | bool | None
+type JSONPrimitive = str | int | float | bool | None
```

Comment:
Is it the new way of declaring a type alias or are we doing this to avoid the using the library's class?

Reply:
this is indeed a Python 3.12 feature! the `type` keyword can now be used to declare type aliases, and in particular it makes it easier to reference symbols only defined later so you don't need to wrap them in a string! The added benefits are of course that the syntax is less verbose and less imports indeed.